tests/test_breath_detection.py

```diff
@@ -285,6 +287,30 @@ def test_remove_no_breaths_around_valid_data():
     )
 
 
+@pytest.mark.parametrize("obj", ["", 1, (1,), []])
+def test_pass_invalid(obj: Any):  # noqa: ANN401
+    bd = BreathDetection(1)
+    with pytest.raises(NotImplementedError):
+        bd.find_breaths(obj)
+
+
+def test_pass_continuousdata(draeger1: Sequence):
+    draeger1 = copy.deepcopy(draeger1)  # prevents writing results to original file
```

Comment:
is there also a test to check for inputting `ContinuousData` (with and without a secondary `Sequence` object?

Reply:
That is this test... First part (302--305) is without sequence, second part (307--311) is with.

Reply:
ah, I was confused, because the type-hint implied that a `Sequence` is passed.